configure.ac

```diff
@@ -40,7 +40,7 @@ MONO_VERSION_BUILD=`echo $VERSION | cut -d . -f 3`
 # This can be reset to 0 when Mono's version number is bumped
 # since it's part of the corlib version (the prefix '1' in the full
 # version number is to ensure the number isn't treated as octal in C)
-MONO_CORLIB_COUNTER=25
+MONO_CORLIB_COUNTER=26
```

Comment:
why is this needed?

Reply:
You need to make sure to have the latest monolite, otherwise you might be missing `Mono.Compiler`